android/src/main/java/com/red5pro/reactnative/stream/R5StreamPublisher.java

```diff
@@ -704,6 +723,92 @@ protected void emitEvent (String type, WritableMap map) {
 
 	}
 
+	protected String getFocusModeAuto () {
+
+		if (mCamera == null || mCamera.getCamera() == null) {
+			return null;
+		}
```

Comment:
I would actually send the `mCamera` (maybe `params` too?) as an argument of getFocusModeAuto() so that function does not use the object state. Thoughts?

Reply:
we will fix this in another PR